llvm/test/Transforms/SLPVectorizer/X86/pr98978.ll

```diff
@@ -0,0 +1,103 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -passes=slp-vectorizer < %s | FileCheck %s
+
+target triple = "x86_64-redhat-linux-gnu"
+
+; Should not get vectorized.
```

Comment:
Would probably be helpful to include a bit more info about why it shouldn't get vectorized, as the test case isn't super easy to read.